client/src/layouts/Home.jsx

```diff
@@ -10,8 +12,8 @@ export default function Home() {
       <h3>Header welcome Page</h3>
 
       <Outlet />
-
-      <footer>footer</footer>
+      <Join />
```

Comment:
first thing, You work a good way. but the best way to do that, create the page name `Home` and create a route for that, and remove it from here.
```suggestion
     
```
and create a page named `Home or Landing or starter, ...`  and create the route in `App.js`

```jsx
function App() {
  return (
    <Routes>
      {/* Routes For Site Views {login, gym filter,...} */}
                            {/* <Home /> for layout*/}
      <Route path="/" element={<Home />}>
        <Route index element={<HomePage />} />
        <Route path="login" element={<LoginPage />} />
        {/* ...  
             other code in this file ... 
        */}
```
and in `Pages/HomePage.js` 
```jsx
import React from "react";

function HomePage() {
  return <div className="container">
              {/* Slider ... */}
             
             {/* Top gyms ... */}

             {/*  <Join />  Join... */}
            {/*  and other section in  Home page ... */}
</div>;
}

export default HomePage;

```

put all sections in the **`new HomePage`**, not in **`Home Layout ,`** but put `Nav bar and footer in Home Layout` because it will use in all page not in Home

### HomePage Sections 

![image](https://user-images.githubusercontent.com/38624002/167140646-ceada84d-84ff-438d-8fd8-409f80320ff8.png)

![image](https://user-images.githubusercontent.com/38624002/167140811-a7b1366f-ceb4-415e-8890-463654ed0113.png)

![image](https://user-images.githubusercontent.com/38624002/167140901-e4a6ad43-ee8d-45fb-84cd-9ca463b381d9.png)



![image](https://user-images.githubusercontent.com/38624002/167141262-4d7a65e4-95cd-4c29-b00d-31422ec850a9.png)